oremda/typing/__init__.py

```diff
@@ -89,15 +90,15 @@ def data(self, data: DataType):
 
 class Port(BaseModel):
     meta: Optional[MetaType] = None
-    data: Optional[DataArray] = None
+    data: Any = None
```

Comment:
Maybe `Optional[Any]` would be better to indicate it could be `None`. Same thing below